token-swap/src/index.ts

```diff
@@ -1,2 +1,120 @@
-import * as ts from '@solana/spl-token-swap';
-import * as w3 from '@solana/web3.js'
+import { Numberu64, TokenSwap } from "@solana/spl-token-swap";
+import { Connection, PublicKey, Account } from "@solana/web3.js";
+
+/**
+ * An Orca token-swap pool
+ * @param address The pool account address
+ * @param nonce The nonce used to generate the pool authority
+ * @param authority The pool authority PDA address to sign instructions
+ * @param poolTokenAddress Public address for the pool token
+ * @param feeAccount Public address of the pool token fee account
+ * @param tokenNameA Name of Token A
+ * @param tokenAccountA Public address of Token A within this pool
+ * @param tokenNameB Name of Token B
+ * @param tokenAccountB Public address of Token B within this pool
+ * @param curveType Trading curve type. 0 - ConstantProduct, 1 - ConstantPrice, 3 - Offset
```

Comment:
I think it's worth adding 2 – Stable in here as well. We're hoping to have the stable curve implementation finished and deployed in the next 2 weeks or so!

Reply:
Sounds good! Let me add that.